### PR TITLE
Documenting support for email

### PR DESCRIPTION
**Pull request by MargotLepizzera**
### What does this PR do?

- Changing secure credentials to global variables
- Add details about email support
- Update screenshots accordingly

### Motivation

PM request

### Preview link

https://docs-staging.datadoghq.com/margot.lepizzera/browser_test_email_support/synthetics/browser_tests/

### Additional Notes
/

### CODE REVIEW

File: content/en/synthetics/browser_tests.md [hunk 1]
```diff
@@ -92,9 +94,8 @@ Advanced options for assertions allow you to specify an X-path or a CSS class or
 The navigation action allow you to:
 
 * Refresh the current page of the scenario.
-* Follow a specific link.
-
-**Note**: In the "Enter link URL" box, users must prepend URLs with `http` or `https`.
+* Follow a specific link. In the "Enter link URL" box, users must prepend URLs with `http` or `https`.
```
Comment (l0k0ms) [inline content/en/synthetics/browser_tests.md]:
```suggestion
* Follow a specific link. In the "Enter link URL" box, you must prepend your URLs with `http` or `https`.
```

File: content/en/synthetics/browser_tests.md [hunk 2]
```diff
@@ -92,9 +94,8 @@ Advanced options for assertions allow you to specify an X-path or a CSS class or
...
+* Go to an email and click link. After creating the [email variable][9], and recording the step that will trigger the email, that step will allow you to access your Synthetics inbox. You then need to choose the email you're interested and click the link you want your browser test to click on.
```
Comment (l0k0ms) [inline content/en/synthetics/browser_tests.md]:
```suggestion
* Click a link from within an email. This step allows you to access your Synthetics mail inbox after creating an [email variable][9] to choose the email you are interested in and click the link you want your browser test to click on.
```

File: content/en/synthetics/browser_tests.md [hunk 3]
```diff
@@ -120,6 +121,8 @@ To create a variable, first give it a name then define its value from:
 | `{{ alphabetic(n) }}`   | Generates an alphabetic string with n letters.      |
 | `{{ alphanumeric(n) }}` | Generates an alphanumeric string with n characters. |
 
+* **An Email**: Generate a random Synthetics email address that you will use in your steps to assert on the email that was sent or to perform actions on the sent email (eg. click confirmation link).
```
Comment (l0k0ms) [inline content/en/synthetics/browser_tests.md]: You should put this above the "A pattern" entry, in order to not have the table messing with the bullet list layout.

Comment (l0k0ms) [inline content/en/synthetics/browser_tests.md]:
```suggestion
* **An Email**: Generate a random Synthetics email address that can be used in your test steps to assert if an email was correctly sent or to perform actions over the sent email content (e.g. click a confirmation link).
```

File: content/en/synthetics/browser_tests.md [hunk 4]
```diff
@@ -73,17 +73,19 @@ Tests can be only recorded from **[Google Chrome][7]**. To record your test, dow
...
+| `Test an element's content`                            | Select any element and check if it contains a specific value. For instance, you could select a `div` and check whether it contains the word "hello".                                                        |
+| `Test that some text is present on the active page` | Asserts that some specific text is present on the current page.                                                                  |
+| `Assert that some text is not present on the active page`          | Asserts that some specific text is **NOT** present on the current page.                                                          |
+| `Test the content of the URL of the active page`                         | This takes the URL of the last page that was interacted with, then asserts whether a specific value (`string`, `number`, `regex`) is present within it. |
+| `Test that an email was received`                         | This asserts that an email was sent, and whether specific values (`string`, `number`, `regex`) are present within the subject or the body of the email. This assertion leverages the [email variable][9]. |
```
Comment (l0k0ms) [inline content/en/synthetics/browser_tests.md]:
```suggestion
| `Test that an email was received`                         | This asserts that an email was sent and whether specific values (`string`, `number`, `regex`) are present within the email subject or body. This assertion leverages  [email variables][9]. |
```

File: content/en/synthetics/browser_tests.md (hunk 3 above)
Comment (MargotLepizzera) [reply content/en/synthetics/browser_tests.md]: Done!

File: content/en/synthetics/browser_tests.md [hunk 5]
```diff
@@ -73,17 +73,19 @@ Tests can be only recorded from **[Google Chrome][7]**. To record your test, dow
 ### Actions
 #### Assertion
 
-{{< img src="synthetics/browser_tests/assertions_browser_check.png" alt="Assertion browser test" responsive="true" style="width:40%;">}}
+{{< img src="synthetics/browser_tests/browser_test_assertions.png" alt="Assertion browser test" responsive="true" style="width:40%;">}}
 
-Assertions allow you to check if an element, a content, or some text is available in the current page. The default timeout for each step is approximately 60 seconds. You can override the timeout for `Assert that an element is present on the page` by expanding the step and changing the `Timeout` value (in seconds).
+Assertions allow you to check if an element, a content, or some text is available in the current page. You can also check if a specific email was sent.
```
Comment (cswatt) [inline content/en/synthetics/browser_tests.md]:
```suggestion
Assertions allow you to check whether an element, some content, or some text is available on the current page. You can also check whether a specific email was sent.   
```

File: content/en/synthetics/browser_tests.md [hunk 6]
```diff
@@ -73,17 +73,19 @@ Tests can be only recorded from **[Google Chrome][7]**. To record your test, dow
...
+| `Test an element's content`                            | Select any element and check if it contains a specific value. For instance, you could select a `div` and check whether it contains the word "hello".                                                        |
+| `Test that some text is present on the active page` | Asserts that some specific text is present on the current page.                                                                  |
+| `Assert that some text is not present on the active page`          | Asserts that some specific text is **NOT** present on the current page.                                                          |
+| `Test the content of the URL of the active page`                         | This takes the URL of the last page that was interacted with, then asserts whether a specific value (`string`, `number`, `regex`) is present within it. |
+| `Test that an email was received`                         | This asserts that an email was sent and whether specific values (`string`, `number`, `regex`) are present within the email subject or body. This assertion leverages  [email variables][9]. |
```
Comment (cswatt) [inline content/en/synthetics/browser_tests.md]:
```suggestion
| `Test that an email was received`                         | Asserts that an email was sent and whether specific values (`string`, `number`, `regex`) are present within the email subject or body. This assertion leverages  [email variables][9]. |
```

File: content/en/synthetics/browser_tests.md [hunk 7]
```diff
@@ -73,17 +73,19 @@ Tests can be only recorded from **[Google Chrome][7]**. To record your test, dow
...
-| `Assert that some text is present anywhere on the page` | Asserts that some specific text is present on the current page.                                                                  |
-| `Assert that some text is nowhere on the page`          | Asserts that some specific text is **NOT** present on the current page.                                                          |
-| `Check main page URL's content`                         | This takes the URL of the last page that was interacted with, then asserts whether a specific value (`string`, `number`, `regex`) is present within it. |
+| `Test that an element is present on the active page`         | Asserts that an element (such as a specific `span`, `div`, `h`, `a`, etc.) is present on the current page.                       |
+| `Test an element's content`                            | Select any element and check if it contains a specific value. For instance, you could select a `div` and check whether it contains the word "hello".                                                        |
```
Comment (cswatt) [inline content/en/synthetics/browser_tests.md]:
```suggestion
| `Test an element's content`                            | Selects an element and checks if it contains a specific value. For instance, you could select a `div` and check whether it contains the word "hello".                                                        |
```

File: content/en/synthetics/browser_tests.md [hunk 8]
```diff
@@ -73,17 +73,19 @@ Tests can be only recorded from **[Google Chrome][7]**. To record your test, dow
...
+| `Test that some text is present on the active page` | Asserts that some specific text is present on the current page.                                                                  |
+| `Assert that some text is not present on the active page`          | Asserts that some specific text is **NOT** present on the current page.                                                          |
+| `Test the content of the URL of the active page`                         | This takes the URL of the last page that was interacted with, then asserts whether a specific value (`string`, `number`, `regex`) is present within it. |
```
Comment (cswatt) [inline content/en/synthetics/browser_tests.md]:
```suggestion
| `Test the content of the URL of the active page`                         | Takes the URL of the last page that was interacted with, then asserts whether a specific value (`string`, `number`, `regex`) is present within it. |
```

File: content/en/synthetics/browser_tests.md [hunk 9]
```diff
@@ -92,9 +94,8 @@ Advanced options for assertions allow you to specify an X-path or a CSS class or
 The navigation action allow you to:
 
 * Refresh the current page of the scenario.
-* Follow a specific link.
-
-**Note**: In the "Enter link URL" box, users must prepend URLs with `http` or `https`.
+* Follow a specific link. In the "Enter link URL" box, you must prepend your URLs with `http` or `https`.
+* Go to an email and click link. This step allows you to access your Synthetics mail inbox after creating an [email variable][9] to choose the email you are interested in and click the link you want your browser test to click on.
```
Comment (cswatt) [inline content/en/synthetics/browser_tests.md]:
```suggestion
* Go to an email and click on a link. This step allows you to access your Synthetics mail inbox after creating an [email variable][9]. Choose the email you are interested in and click the link you want your browser test to click on.
```

**Pull request merged**